variables.tf

```diff
@@ -84,19 +84,24 @@ variable "enable_http_endpoint" {
 
 variable "engine" {
   type        = string
-  default     = "aurora"
+  default     = "aurora-mysql"
```

Comment:
why change the default to aurora-mysql?

Reply:
Mainly because it's a drama to find the defaults. So `aurora-mysql` is the "new" MySQL V2 version of serverless (which maps to `5.7.mysql_aurora.2.08.3`). See above

Reply:
I get your point, but then we should automate the update of this variable. But since this comment is out of scope of your PR, no comments about this anymore